src/main/flight/pid.c

```diff
@@ -107,13 +112,47 @@ PG_RESET_TEMPLATE(pidProfile_t, pidProfile,
     .D8[PIDVEL] = 1,
 
     .yaw_p_limit = YAW_P_LIMIT_MAX,
-    .dterm_lpf = 100,   // DTERM filtering ON by default
-    .yaw_lpf = 80,
+    .yaw_lpf_hz = 0,
+    .dterm_filter_type = FILTER_BIQUAD,
+    .dterm_lpf_hz = 100,    // filtering ON by default
+    .dterm_notch_hz = 260,
+    .dterm_notch_cutoff = 160,
     .deltaMethod = PID_DELTA_FROM_MEASUREMENT,
     .horizon_tilt_effect = 75,
     .horizon_tilt_mode = HORIZON_TILT_MODE_SAFE,
 );
 
+float getdT(void)
+{
+    static float dT;
+    if (!dT) dT = (float)targetPidLooptime * 0.000001f;
```

Comment:
good, as the testing impact this had was problematic.  I'll take a closer look at this now the tests are passing again.